app/celery/tasks.py

```diff
@@ -46,6 +47,7 @@
 from app.models import (
     Job,
     Notification,
+    DVLA_STATUS_SENT,
```

Comment:
There is already JOB_STATUS_SENT_TO_DVLA if this is different might be worth expanding it? I prefer your one so could consider changing the existing one?

Reply:
I think that's something different, this is handling letters at the notification level, not job

Reply:
Maybe a rename to make it clearer? We should probably prefix all the notification statuses and all the job status with something? Or use two enums JobStatus.Sent and NotificationStatus.Sent?